docs/pages/inboxes/build-inbox.mdx

```diff
@@ -26,202 +26,189 @@ Learn to build a chat inbox with the help of [XMTP.chat](https://xmtp.chat/), an
 
-  ```tsx [Browser]
-  import type { Signer } from "@xmtp/browser-sdk";
+  ```tsx [Browser] 
+  import type { Signer } from "@xmtp/browser-sdk"; 
 
-  const accountAddress = "0x...";
+  const accountIdentity = { 
+    kind: "ETHEREUM", // Specifies the identity type 
+    identifier: "0x...", // Ethereum address as the identifier 
+  }; 
 
-  const signer: Signer = {
-    getAddress: () => accountAddress,
-    signMessage: async (message) => {
-      // return value from a signing method here
-    },
-  };
+  const signer: Signer = { 
+    getIdentity: () => accountIdentity, 
+    signMessage: async (message) => { 
+      // return value from a signing method here 
+    }, 
+  }; 
   ```
 
-  ```tsx [Node]
-  import type { Signer } from "@xmtp/node-sdk";
+  ```tsx [Node] 
+  import type { Signer } from "@xmtp/node-sdk"; 
 
-  const accountAddress = "0x...";
+  const accountIdentity = { 
+    kind: "ETHEREUM", // Specifies the identity type 
+    identifier: "0x...", // Ethereum address as the identifier 
+  }; 
 
-  const signer: Signer = {
-    getAddress: () => accountAddress,
-    signMessage: async (message) => {
-      // return value from a signing method here
-    },
-  };
+  const signer: Signer = { 
+    getIdentity: () => accountIdentity, 
+    signMessage: async (message) => { 
+      // return value from a signing method here 
+    }, 
+  }; 
   ```
 
-  ```tsx [React Native]
-  // Example EOA
-  export function convertEOAToSigner(eoaAccount: EOAAccount): Signer {
-    return {
-      getAddress: async () => eoaAccount.address,
-      signMessage: async (message: string | Uint8Array) =>
-        eoaAccount.signMessage({
-          message: typeof message === "string" ? message : { raw: message },
-        }),
-      walletType: () => undefined, // Default: 'EOA'
-      getChainId: () => undefined,
-      getBlockNumber: () => undefined,
-    };
-  }
+  ```tsx [React Native] 
+  // Example EOA Signer 
+  export function convertEOAToSigner(eoaAccount: EOAAccount): Signer { 
+    return { 
+      getIdentity: async () => ({ 
+        kind: "ETHEREUM", // Specifies the identity type 
+        identifier: eoaAccount.address, // Ethereum address as the identifier 
+      }), 
+      signMessage: async (message: string | Uint8Array) => 
+        const signature = eoaAccount.signMessage({ 
+          message: typeof message === "string" ? message : { raw: message }, 
+        }), 
+        
+       return {
+        signature,
+      }
+      getChainId: () => undefined, 
+      getBlockNumber: () => undefined, 
+    }; 
+  } 
   ```
 
-  ```kotlin [Kotlin]
-  class EOAWallet : SigningKey {
-    override val address: String
-        get() = walletAddress
-
-    override suspend fun sign(message: String): Signature {
-  		val signature = key.sign(data)
-  		return signature
-    }
-
-    override suspend fun sign(data: ByteArray): Signature {
-  		val signature = key.sign(message: message)
-  		return signature
-    }
+  ```kotlin [Kotlin] 
```

Comment:
As requested, updated to match the updated Swift sample

Reply:
Pushed commit for this